packages/fast-components-styles-msft/src/toggle/index.ts

```diff
@@ -0,0 +1,151 @@
+import {IDesignSystem, hexToRGB} from "../design-system";
+import {ComponentStyles} from "@microsoft/fast-jss-manager";
+import {IToggleClassNameContract} from "@microsoft/fast-components-class-name-contracts";
+
+const styles: ComponentStyles<IToggleClassNameContract, IDesignSystem> = {
+    toggle: {
+        '& > label': {
+            display: 'inline-block',
+            fontSize: '13px',
+            lineHeight: '20px',

```

Comment:
For all these pixel values, you can import a utility:

`import {toPx} from "../utilities/units";`

Reply:
Wondering why?
Why is it better to send a number and import a utility instead of just setting it correctly?

Reply:
I'll change as I see this being done everywhere, but I would still like to understand usage.